.bazelci/tests.yml

```diff
@@ -89,15 +89,20 @@ tasks:
     name: rolling_ubuntu
     <<: *ubuntu
     <<: *rolling
+  rolling_ubuntu_bzlmod:
+    name: rolling_ubuntu_bzlmod
+    platform: ubuntu1804
+    build_flags:
+      - "--enable_bzlmod"
+    <<: *common
+    <<: *rolling
+    <<: *default_tests
   rolling_macos:
-    name: rolling_macos
+    name: last_green_macos
     # It seems there is no rolling Bazel for macos.
```

Comment:
Double check this maybe? I'm pretty sure I got a rolling macOS last week.

Reply:
It does work now.